tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -229,6 +229,21 @@ if (tensorflow_ENABLE_GPU)
   endif()
 endif()
 
+#  Resolve the installed version of Python (for Python.h and python).
+include(FindPythonInterp)
```

Comment:
You're right, it should be enough. Then we can check if(${PYTHONINTERP_FOUND}) (https://cmake.org/cmake/help/v3.0/module/FindPythonInterp.html). I'll draft up a new commit.